packages/rn-tester/NativeComponentExample/js/MyNativeView.js

```diff
@@ -255,6 +247,19 @@ export default function MyNativeView(props: {}): React.Node {
       <Text style={{color: 'green', textAlign: 'center'}}>
         InLayout {getTextFor(legacyMeasureLayout)}
       </Text>
+      <Button
+        title="Fire Legacy Style Event"
+        onPress={() => {
+          RNTMyNativeViewCommands.fireLagacyStyleEvent(
+            // $FlowFixMe[incompatible-call]
+            ref.current,
+          );
+          setLegacyStyleEventCount(prevCount => prevCount + 1);
```

Comment:
we should call `setLegacyStyleCount` when the event is received by JS. It should be line 126. Otherwise we are not actually tested that the event is sent and received back.

Reply:
done